tools/review/get_comments_markdown.py

```
@@ -0,0 +1,296 @@
+
+    # Determine owner/repo precedence:
+    # 1) --remote / remote=  2) --repo-slug / repo=owner/repo  3) --owner/--repo or owner=/repo=
+    # 4) if inside git repo: origin remote  5) GITHUB_REPOSITORY env
+    owner = None
+    repo = None
+
+    remote_arg = args.remote or kv.get("remote")
+    slug_arg = args.slug or kv.get("slug")
+    repo_kv = kv.get("repo")
+    owner_arg = args.owner or kv.get("owner")
+    repo_name_arg = args.repo or (kv.get("repo_name") if repo_kv and "/" not in repo_kv else None)
```

Ruff (0.14.3)

Local variable `repo_name_arg` is assigned to but never used

Remove assignment to unused variable `repo_name_arg`

(F841)